app/js/controllers/crafterstats.js

```diff
@@ -45,7 +45,7 @@
 
     function crafterActionClasses(action, cls) {
       return {
-        'faded': !$scope.isActionSelected(action, cls),
+        'faded': !$scope.isActionSelected(action, cls)
```

Comment:
Regarding code style, I prefer to retain trailing commas where they are valid, since it makes adding new elements easier in the future.